lib/slide_rule/distance_calculators/day_of_year.rb

```diff
@@ -3,15 +3,15 @@ module DistanceCalculators
     class DayOfYear
       DAYS_IN_YEAR = 365
 
-      def calculate(date_1, date_2)
+      def calculate(date_1, date_2, options={})
         date_1 = cleanse_date(date_1)
         date_2 = cleanse_date(date_2)
-
+        threshold = options[:threshold] || DAYS_IN_YEAR
```

Comment:
I was thinking about this earlier, but forgot to mention. I wonder if we should call this option something like `max_days`. `threshold` is a little too generic. I could see this options array going a couple of different ways. We could make the individual options named very specific to how they are used by the calculator, or we can try to come up with some general names that are used by many calculators (like rubocop). ??